### PR TITLE
Fourth Commit. Fixed bugs specified in trello.

### PR DESCRIPTION
**Pull request by OnlyAlf**

### CODE REVIEW

**Review by itzcoatl90 — CHANGES_REQUESTED** [review]
merge Development to yours and push or rebase please

File: app/src/main/res/layout/activity_update_password.xml [hunk 1]
```diff
@@ -10,34 +10,53 @@
...
     android:paddingBottom="@dimen/activity_vertical_padding">
 
-    <android.support.design.widget.TextInputEditText
-        android:id="@+id/et_update_password"
+
+    <android.support.design.widget.TextInputLayout
+        android:id="@+id/ti_update_retype"
         android:layout_width="match_parent"
         android:layout_height="@dimen/te_button_height"
+
```
Comment (itzcoatl90) [inline app/src/main/res/layout/activity_update_password.xml:18]: Please remove empty lines of code for making your PR clean

**Review by itzcoatl90 — CHANGES_REQUESTED** [review]
Sorry, branch keep braking the app

**Comment by itzcoatl90** [comment]
screen shot 2018-07-21 at 4 26 34 pm

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (itzcoatl90) [inline .idea/codeStyles/Project.xml:1]: This file should not be in PR

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (itzcoatl90) [inline .idea/vcs.xml:1]: This file should not be in PR

**Review by itzcoatl90 — DISMISSED** [review]
please

**Review by ecejmsr — APPROVED** [review]
Everything looks fine.

**Pull request merged**